OpenNefia.Content/UI/Hud/IHudLayer.cs

```diff
@@ -1,9 +1,15 @@
-﻿using OpenNefia.Core.UI.Layer;
+﻿using OpenNefia.Core.Maths;
+using OpenNefia.Core.UI.Layer;
 
 namespace OpenNefia.Content.UI.Hud
 {
     public interface IHudLayer : IUiLayer
     {
         public IHudMessageWindow MessageWindow { get; }
+        public UIBox2i GameBounds { get; }
+        void ToggleBacklog(bool visible);
```

Comment:
I don't think this should be in `IHudLayer` since it's going to become a manager for all kinds of HUD widgets, including ones added by mods, so we can't assume the HUD will always have a "backlog" widget.

Reply:
changed in 9eb449f